buildtools/pipenv/pipenv.go

```diff
@@ -55,23 +55,23 @@ func (p Cmd) Deps() (graph.Deps, error) {
 }
 
 // GraphJSON returns the output from `pipenv graph --json-tree`.
-func GraphJSON(dirname string) (string, error) {
+func GraphJSON(dirname string) (string, *errors.Error) {
 	out, _, err := exec.Run(exec.Cmd{
 		Name: "pipenv",
 		Argv: []string{"graph", "--json-tree"},
 		Dir:  dirname,
 	})
 	if err != nil {
-		err = errors.Wrap(err, "Could not run `pipenv graph --json-tree` within the current directory")
+		return "", errors.UnknownError(err, "Could not run `pipenv graph --json-tree` within the current directory")
```

Comment:
Suggestion... Say "the current directory" only if dirname is "." but otherwise state the dirname itself.

Reply:
Will do, planning to use `pip.go` as a demo for what the new errors package works before extrapolating out to the rest of the code. I think this is out of scope for this PR but I do like your point.